routes/socket/game/common.js

```diff
@@ -187,7 +187,24 @@ module.exports.startElection = (game, specialElectionPresidentIndex) => {
 			() => {
 				if (game.gameState.timedModeEnabled) {
 					const chancellorIndex = _.shuffle(game.gameState.clickActionInfo[1])[0];
-
+					game.chats.push({
+						gameChat: true,
+						timestamp: new Date(),
+						chat: [
+							{
+								text: 'The timer has forced '
+							},
+							{
+								text: game.general.blindMode
+									? `${replacementNames[presidentIndex]} {${presidentIndex + 1}} `

```

Comment:
`replacementNames` is not defined here, please use `game.general.replacementNames`!

And also test blind games 😉

Reply:
thanks, believe i have solved this